app/data/get-queries.js

```diff
@@ -20,7 +20,11 @@ const getQueries = async (sbi) => {
       payload: JSON.stringify({ query }),
       json: true
     })
-    return payload.data.customerQueryTicketsBySbi.customerQueryTickets
+
+    const queries = payload.data.customerQueryTicketsBySbi.customerQueryTickets
+    queries.sort((a, b) => new Date(b.timestamp) - new Date(a.timestamp))
```

Comment:
Have added this in so that the queries for the external user are arranged in descending order by `timestamp` i.e. latest queries are shown at the top.